src/editor/viewport/propertyManager/boxProperty.js

```diff
@@ -1,9 +1,10 @@
 import PropertyController from "./propertyController";
 
 export default class BoxProperty extends PropertyController{
-    constructor(propertyPane, geometry){
+    constructor(propertyPane, geometry, material){
         super(propertyPane);
         this.geometry = geometry;
+        this.material = material;
```

Comment:
u have used material in PropertyController's constructor, then y don't uh pass it in super()